test/e2e/kubectl/portforward.go

```diff
@@ -32,9 +32,13 @@ import (
 	"syscall"
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
nit: can you group these with the other imports below?

Reply:
thx for review, fixed, PTAL